mesa_geo/geospace.py

```diff
@@ -1,45 +1,85 @@
+from __future__ import annotations
+from typing import List, Optional, Union, Tuple
+import copy
+import math
+import warnings
+
 import pyproj
 from libpysal import weights
 from rtree import index
 from shapely.geometry import Point
 from shapely.prepared import prep
+import numpy as np
+import geopandas as gpd
+from affine import Affine
+import rasterio as rio
+from rasterio.warp import (
+    calculate_default_transform,
+    transform_bounds,
+    reproject,
+    Resampling,
+)
 
 from mesa_geo.geoagent import GeoAgent
 
 
 class GeoSpace:
-    def __init__(self, crs="epsg:3857"):
+    def __init__(self, crs="epsg:3857", **kwargs):
```

Comment:
Why is this `**kwargs` instead of just explicitly adding `warn_crs_conversion`?

Reply:
I think [this comment](https://github.com/projectmesa/mesa-geo/pull/67#issuecomment-1133533502=) means just a single keyword argument.

Reply:
@Corvince Would you like to clarify? I'm a bit confused whether it's a single keyword argument or `kwargs` here.

Reply:
I get it! `warn_crs_conversion` is now a keyword-only argument with default value of `True`.